lib/miq_expression/tag.rb

```diff
@@ -19,11 +20,12 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
-    @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
+    @namespace = managed ? MANAGED_NAMESPACE : USER_NAMESPACE
+    @path = "/#{namespace}/#{column}"
```

Comment:
I think looking at your other comments (b) might be more appropriate here, at least for now?